lib/resqued/listener.rb

```diff
@@ -32,7 +32,7 @@ def exec
       socket_fd = @socket.to_i
       ENV['RESQUED_SOCKET']      = socket_fd.to_s
       ENV['RESQUED_CONFIG_PATH'] = @config_paths.join(':')
-      ENV['RESQUED_STATE']       = (@old_workers.map { |r| "#{r[:pid]}|#{r[:queue]}" }.join('||'))
+      ENV['RESQUED_STATE']       = (@old_workers.map { |r| "#{r[:pid]}|#{r[:queue_key]}" }.join('||'))
```

Comment:
Renaming the hash key here is not really necessary, but I thought it might make things clearer.

Reply:
👍 on renaming it. There's one more place where it's needed:

https://github.com/spraints/resqued/blob/6850bc51271b95d75c4b44c2718c25364f60f4eb/lib/resqued/listener_proxy.rb#L58-L61

Reply:
Nicely spotted! I fixed it with 0edda2963fa8a89fb19f2ff557698bb5760473df
There should be no other occurrence of that Symbol:
```sh
$ git grep ':queue[^s_]'

lib/resqued/listener_proxy.rb:60:      worker_pids.map { |pid, queue| { :pid => pid, :queue => queue } }
```